src/components/List.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import {Crypto} from './Interfaces';
+
+export const List = ({cryptos} /* :Crypto[] */) => {
+  let bull: boolean;
+
+  return (
+    <>
+      {cryptos.map((el: Crypto) => {
+        return (
+          <>
+            <View style={[styles.cryptoContainer, {padding: 25}]}>
+              <View style={styles.cryptoContainer}>
+                <Image style={styles.image} source={el.img} />
+                <View style={[{paddingLeft: 5}]}>
+                  <Text style={styles.name}> {el.name} </Text>
+                  <Text> {el.symbol} </Text>
+                </View>
+              </View>
+              <View style={styles.lineContainer}>
+                <View>
+                  <Text style={styles.name}> ${el.price} </Text>
+                  <View style={styles.percentContainer}>
+                    {(bull = el.percent > 0 ? true : false)}
+                    {/* { bull = el.percent > 0? arrow = require('../Images/ArrowUp.png') : arrow = require('../Images/ArrowDown.png') } */}
+                    <Image style={[{width: 15, height: 15}]} source={el.arrow} />
+                    <Text
+                      style={[
+                        {
+                          textAlign: 'right',
+                          color: bull ? 'green' : 'red',
+                          width: 'auto',
+                        },
+                      ]}>
+                      {' '}
+                      {Math.abs(el.percent)}%{' '}
```

Comment:
look out for {' '}

Reply:
That's not me, it was the prettier extension